doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -502,9 +502,9 @@ The following instructions guide you through the process of building a macOS **O
 
 ### 1. Prepare your system
 :apple:
-You must install a number of software dependencies to create a suitable build environment on your system:
+You must install a number of software dependencies to create a suitable build environment on your system (the specified versions are minimums):
 
-- [Xcode >= 11.4](https://developer.apple.com/download/more/) (requires an Apple account to log in).
+- [Xcode 10.3](https://developer.apple.com/download/more/) (requires an Apple account to log in).
```

Comment:
I think this should be 11.4.1 which according to [1] is the minimum which supports code signing. Likewise for building Java 15.

[1] https://developer.apple.com/support/code-signing/

Reply:
I've included the 11.4.1 information. Since the OpenJ9 project still builds with 10.3, it seems appropriate to keep this as the minimum. Not all builds are going to be signed.

Reply:
That works for me.